xc7/primitives/common_slice/routing/Nused/CMakeLists.txt

```diff
@@ -1,16 +1,7 @@
 add_file_target(FILE ntemplate.Nused.sim.v SCANNER_TYPE verilog)
```

Comment:
As far as I can tell, the `Nused` elements don't have any features / functionality? They don't seem to appear in any of the XML files?